deploy/helm/omni/values.yaml

```diff
@@ -0,0 +1,68 @@
+
+nameOverride: omni  # optional, if ommitted the Chart name will be used
+debug: false
+domainName: omni.makomedia.nl
+accountUuid: D5C2A21E-7162-4122-B923-A87ED7827F5B
+privateKeySource: "/omni.asc"
+deployment:
+  image: ghcr.io/siderolabs/omni:latest
+  replicaCount: 1
+  annotations:
+    amazing: Annotation
+  imagePullPolicy: IfNotPresent
+auth:
+  auth0:
+    enabled: true
+    clientId: 123456
+    domain: "https://www.auth0.example"
+  saml:
+    enabled: false
+    url: "https://www.saml.example"
+initialUsers:
+ - foo@bar.com
+service:
+  type: ClusterIP # currently only ClusterIP is supported by this Helm Chart
+  api:
+    port: 443
+    targetPort: 443
+  siderolink:
+    api:
+      bindAddress: 0.0.0.0
+      port: 8090
+      targetPort: 8090
+    wireguard: 
+      port: 50180
+      targetPort: 50180
+      address: 192.168.68.107
```

Comment:
Where does this address come from?

Reply:
Agreed, this is unclear. Removed the address and inserted a comment to help the user figure out what to put here.